manifests/cron.pp

```diff
@@ -0,0 +1,8 @@
+class centrify::cron {
+  cron { 'flush-centrify-cache':
+    ensure  =>  present,
+    command =>  '/usr/sbin/adflush -f > /dev/null 2>&1',
+    user    =>  root,
+    minute  =>  '*/30',
```

Comment:
Could the cron times be made configurable? I think if this is added, the times should be configurable and the default should be something like once a day.